scene/gui/tree.cpp

```diff
@@ -1722,7 +1739,7 @@ void Tree::draw_item_rect(TreeItem::Cell &p_cell, const Rect2i &p_rect, const Co
 
 	if (rtl) {
 		Point2 draw_pos = rect.position;
-		draw_pos.y += Math::floor((rect.size.y - p_cell.text_buf->get_size().y) / 2.0);
+		draw_pos.y += Math::floor(p_draw_ofs.y) - _get_title_button_height();
```

Comment:
Should this really use title height? It's a method for drawing cells (I think) and titles can be disabled, making their height 0.

Reply:
The title height is used for the initial drawing offset. If the title is hidden, then there is no offset. Perhaps this value is already taken into account in some variable, but I could not fix the offset error until I used this function. The `Tree` class is very large and needs to be refactored.